pyaedt/mechanical.py

```diff
@@ -37,23 +10,58 @@
 
 
 class Mechanical(FieldAnalysis3D, object):
-    """Mechanical class.
+    """Mechanical instance interface.
+
+    This class exposes Mechanical functionalities pythonically.
```

Comment:
Use of ``pythonically`` seems jargony. Rewrite?

Reply:
Agreed.  Rewrite to:
This class exposes features from the mechanical application.